### ISSUE TITLE
Setup a console command for clearing entry versions?

### ISSUE THREAD

**Issue by aaronbushnell** [carlcs/craft-deleteallentryversions#4]
Thanks for your work on this awesome plugin, @carlcs! I have a couple sites with a pretty substantial amount of versions that need to be cleared out. It'd be awesome if I could run this via the Craft console so things like `max_execution_time` and `memory_limit` are bypassed. And so it's easier to trigger them before/after a deploy, etc.

**Comment by zizther** [comment]
+1 for this

**Comment by aaronbushnell** [comment]
Now that Craft 3.2 has support for `maxRevisions` [in the config](https://docs.craftcms.com/v3/config/config-settings.html#maxrevisions) I wonder if this is better handled by Craft itself?

**Comment by zizther** [comment]
This is true. I presume if you set this, it will delete the remaining ones outside the limit when resaving the entry

**Comment by aaronbushnell** [comment]
Correct, @zizther!
